_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -878,6 +878,7 @@
 /area/security/main)
 "acq" = (
 /obj/effect/landmark/secequipment,
+/obj/effect/turf_decal/bot,
```

Comment:
This is **-VERY-** out of place compared to the rest of them. There are no other decals in the department and these are just random spots in the middle of the floor. None of the other lockers have them.

Reply:
Primary reason is to give Security a little hint about where extra lockers will end up spawning. As of right now, there's pretty much no way to know where the extra lockers are gonna end up on each map until you either look at the map in dreammaker, or actually witness a locker spawn in. This can cause a small amount of confusion during highpop, as the lockers seem like they're cropping up in random places as new sec officers arrive.

Reply:
...wait they spawn in as more sec joins? I've never noticed that before.... huh.

Reply:
Done 👍